client/src/components/Search/Filters/Filters.js

```diff
@@ -1,26 +1,46 @@
 import * as React from 'react';
 import './Filters.scss';
 import { withRouter } from 'react-router-dom';
-import { Form } from 'react-bootstrap';
+import { Collapse, Form } from 'react-bootstrap';
 import TagsSearch from '../TagsSearch/TagsSearch';
 import search from "../../../utils/search";
+import DatePicker from 'react-datepicker/es';
```

Comment:
What's the difference of importing `react-datepicker/es` and 'react-datepicker'?

Reply:
Espanol senior.